reduxpractice/src/actions/index.js

```diff
@@ -0,0 +1,22 @@
+ export const add=(data)=>
+{
+   return { type:"ADD",payload:data }     
+}
+
+export const loading=(data)=>
+{
+    return {type:"loading",payload:data}
+}
+
+export const select=(data) =>
+{
+    return {type:"select",payload:data}
+}
+
+export const addAll=(data) =>
```

Comment:
Also i'll recommend using constants for type. so you could use them in reducers as well